newIDE/app/src/UI/QrCode.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import * as React from 'react';
+import QrCreator from 'qr-creator';
+import GDevelopThemeContext from './Theme/GDevelopThemeContext';
+
+type Props = {|
+  url: string,
+  size?: number,
+|};
+
+const QrCode = ({ url, size = 128 }: Props) => {
+  const gdevelopTheme = React.useContext(GDevelopThemeContext);
+
+  const containerRef = React.useRef<?HTMLDivElement>(null);
+  React.useEffect(
+    () => {
+      const containerElement = containerRef.current;
+
+      if (!containerElement) return;
+
+      QrCreator.render(
+        {
+          text: url,
+          radius: 0,
+          ecLevel: 'M', // L, M, Q, H
```

Comment:
reasoning behind this value?

Reply:
It changes the size of the qr code. I'll add a comment.